apps/src/gamelab/PropTypes.js

```diff
@@ -132,17 +132,57 @@ export function throwIfSerializedAnimationListIsInvalid(serializedAnimationList)
     throw new Error('Animation List orderedKeys should be an array');
   }
```

Comment:
Given that [it's all of ten lines of code](https://github.com/dmitriiabramov/react-prop-types-check/blob/master/src/react_prop_types_check.js) I think we could take a similar local approach (much like your tests over your custom prop validators do).  This is a good idea, I'll run with it.